Misc/NEWS.d/next/Library/2025-02-03-16-39-35.gh-issue-129605.sN_0uq.rst

```diff
@@ -0,0 +1 @@
+Fix pyrepl crash when a :exc:`NameError` occurs in the ``__getattr__`` method.
```

Comment:
I think you can use :meth:`~object.__getattr__`.

Reply:
Addressed in c969cca3491